app/forms/umrdr/forms/work_form.rb

```diff
@@ -31,5 +29,14 @@ def rights
         @model.rights.first
     end
 
+    # You like danger?! Because you better pass in a hash with the correct keys. 
+    # Can be alleviated when form coverage attribute is:
+    # `coverages: [{begin: {:year, :month, :day}, end: {:year, :month, :day}}, ... ]`
+    def merge_date_coverage_attributes!(hsh)
+      hsh ||= {}
+      hsh.stringify_keys!
+      @attributes.merge! hsh
+    end
```

Comment:
If you know that you will only ever have `nil`, and not `false` passed in you can also write:

```ruby
  @attributes.merge!(hsh&.stringify_keys || {})
```